flink-runtime/src/main/scala/org/apache/flink/runtime/messages/ExecutionGraphMessages.scala

```diff
@@ -80,7 +81,7 @@ object ExecutionGraphMessages {
       jobID: JobID,
       newJobStatus: JobStatus,
       timestamp: Long,
-      error: Throwable)
+      error: SerializedThrowable)
```

Comment:
Why not giving the serialized throwable a name which indicates that it's actually a serialized throwable?